src/distilabel/steps/base.py

```diff
@@ -132,10 +132,62 @@ def connect(
                 columns will be mapped by name. This is useful when the names of the
                 output columns of the current step are different from the names of the
                 input columns of the other step. Defaults to `None`.
+
+        Returns:
+            The step connected, to allow nested calls to the `connect` method.
         """
         if input_mappings is not None:
             step.input_mappings = input_mappings
         self.pipeline._add_edge(self.name, step.name)  # type: ignore
+        return step
+
+    def __rshift__(self, other: Union["_Step", List["_Step"]]) -> "_Step":
+        """Allows using the `>>` operator to connect steps in the pipeline.
+
+        Args:
+            other: The step to connect to or a list of steps to connect to.
+
+        Returns:
+            The connected step, or the last one of the list of steps if a list is passed.
+
+        Example:
+            ```python
+            step1 >> step2
+            # Would be equivalent to:
+            step1.connect(step2)
+
+            # It also allows to connect a list of steps
+            step1 >> [step2, step3]
+            ```
+        """
+        if isinstance(other, list):
+            for step in other:
+                self.connect(step)
+            return other
+        else:
+            return self.connect(other)
```

Comment:
Opinionated but I think @gabrielmbmb feels the same way

```suggestion
        return self.connect(other)
```